### PR TITLE
EBODAC-474 "QA - Why are hidden buttons in previous deployment now vi…

### PR DESCRIPTION
**Pull request by tomaszmueller**
…sible and enabled? Test"

### CODE REVIEW

**Comment by tomaszmueller** [comment]
test this please

File: ebodac/src/test/java/org/motechproject/ebodac/uitest/test/HiddenButtonsEnabledUiTest.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.motech.page.LoginPage;
+import org.motech.test.TestBase;
+import org.motechproject.ebodac.uitest.page.HomePage;
+import org.motechproject.ebodac.uitest.page.ParticipantEditPage;
+import org.motechproject.ebodac.uitest.page.ParticipantPage;
+
+import static junit.framework.Assert.assertTrue;
```
Comment (pmuchowski) [inline ebodac/src/test/java/org/motechproject/ebodac/uitest/test/HiddenButtonsEnabledUiTest.java]: change it to org.junit.Assert.assertTrue;

File: ebodac/src/test/java/org/motechproject/ebodac/uitest/test/HiddenButtonsEnabledUiTest.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+    @Before
+    public void setUp() throws Exception {
+        l1adminUser = "admin";
+        l1adminPassword = System.getProperty("admin_password");
+        loginPage = new LoginPage(driver);
+        homePage = new HomePage(driver);
+        participantPage = new ParticipantPage(driver);
+        participantEditPage = new ParticipantEditPage(driver);
+        if (homePage.expectedUrlPath() != currentPage().urlPath()) {
```
Comment (pmuchowski) [inline ebodac/src/test/java/org/motechproject/ebodac/uitest/test/HiddenButtonsEnabledUiTest.java]: you should not compare two Strings with "!=" or "==", use equals methods instead or if both strings can be null use org.apache.commons.lang.StringUtils.equals

File: ebodac/src/test/java/org/motechproject/ebodac/uitest/test/HiddenButtonsEnabledUiTest.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    @Test
+    public void hiddenButtonsEnabledTest() throws Exception {
+        assertTrue(homePage.isEBODACModulePresent());
+        assertTrue(homePage.isIVRModulePresent());
+        assertTrue(homePage.isSMSModulePresent());
+        homePage.openEBODACModule();
+        participantPage.openFirstParticipant();
+        assertTrue(participantEditPage.checkButtons());
+
+
```
Comment (pmuchowski) [inline ebodac/src/test/java/org/motechproject/ebodac/uitest/test/HiddenButtonsEnabledUiTest.java]: remove those unnecessary lines

**Comment by tomaszmueller** [comment]
@pmuchowski corrected

**Comment by tomaszmueller** [comment]
test this please

File: ebodac/src/test/java/org/motechproject/ebodac/uitest/test/HiddenButtonsEnabledUiTest.java [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    @Before
+    public void setUp() throws Exception {
+        l1adminUser = "admin";
+        l1adminPassword = System.getProperty("admin_password");
+        loginPage = new LoginPage(driver);
+        homePage = new HomePage(driver);
+        participantPage = new ParticipantPage(driver);
+        participantEditPage = new ParticipantEditPage(driver);
+        if (StringUtils.equals(homePage.expectedUrlPath(), currentPage().urlPath())) {
```
Comment (pmuchowski) [inline ebodac/src/test/java/org/motechproject/ebodac/uitest/test/HiddenButtonsEnabledUiTest.java]: it was "!=" before and now you have equals, should not this be !StringUtils.equals?

**Comment by tomaszmueller** [comment]
corrected

**Comment by tomaszmueller** [comment]
test this please

**Comment by tomaszmueller** [comment]
test this please

**Pull request merged**